benchmarks/replay/replay.cpp

```diff
@@ -45,7 +45,7 @@
 #include <thread>
 
 /// MR factory functions
-std::shared_ptr<rmm::mr::device_memory_resource> make_cuda(std::size_t = 0)
+std::shared_ptr<rmm::mr::device_memory_resource> make_cuda(std::size_t /*unused*/ = 0)
```

Comment:
Is the `/*unused*/` thing required by clang-tidy?

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/readability-named-parameter.html

Originates from Google style guidelines. I am not super fussed about the /*name*/ part, but I do like that this rule also verifies that declaration and implementation use the same names.